docs/conf.py

```diff
@@ -0,0 +1,106 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
This file has lots of commented out lines. Is it possible to reduce / remove them if they are not needed?

Reply:
I suspect they might be needed later! For example, it might be nice to remove (eg.) the `sdk.lusid.rest` module from the docs if there's nothing useful for the customer to see there. I'd quite like to have these ready-to-go unless you think it's a big problem? (otherwise I have to store useful conf settings somewhere...)

Reply:
If you plan to use them then its probably fine to leave them in for now. When you get to a point where you are happy with how we are implementing Sphinx (hopefully sooner, rather than later), we should remove any unnecessary code.